pkg/network/node/ovscontroller.go

```diff
@@ -58,10 +58,10 @@ func (oc *ovsController) AlreadySetUp() bool {
 		return false
 	}
 	expectedVersionNote := oc.getVersionNote()
-	for _, flow := range flows {
-		parsed, err := ovs.ParseFlow(ovs.ParseForDump, flow)
-		if err == nil && parsed.Table == ruleVersionTable && parsed.NoteHasPrefix(expectedVersionNote) {
-			return true
+	for i := len(flows)-1; i >= 0; i-- {
```

Comment:
Adding a comment here will be helpful: Processing the flows in the reverse order will find the flow rule quicker as version table is on the highest numbered table

Reply:
done